packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -236,7 +236,6 @@ class CliServiceProvider
       delete clientOptionsCopy.oidc;
       delete clientOptionsCopy.parentHandle;
       delete clientOptionsCopy.parentState;
```

Comment:
Completely optional, but feel free to add these (since it goes hand in hand with the rest of the devtools-connect update), otherwise I'll do it in a follow-up PR:

```suggestion
      delete clientOptionsCopy.proxy;
      delete clientOptionsCopy.applyProxyToOIDC;
```